src/test/containers/AppliancePageTest.js

```diff
@@ -0,0 +1,219 @@
+    });
+
+    it("Should change pages from unconfigured to configured and vice versa", () => {
+        const wrapper = shallow(<AppliancePage />);
+
+        expect(wrapper.state("pageStateUnconfigured")).to.equal(true);
+
+        wrapper.find("#pageButtonConfigured").simulate("click");
+
+        expect(wrapper.state("pageStateUnconfigured")).to.equal(false);
+
+        wrapper.find("#pageButtonUnconfigured").simulate("click");
+
+        expect(wrapper.state("pageStateUnconfigured")).to.equal(true);
+    });
+
+    it("Should change the number of a page", () => {
+        const wrapper = shallow(<AppliancePage />);
+
+        expect(wrapper.find("#paginationButton")).to.have.lengthOf(2);
+        expect(wrapper.state("showPagination")).to.equal(true);
+        expect(wrapper.state("currentPage")).to.equal(0);
+
+        wrapper
+            .find("#paginationButton")
+            .at(1)
+            .simulate("click");
+
+        expect(wrapper.state("currentPage")).to.equal(1);
+    });
+
+    it("Should update a list of selected appliances", () => {
+        // mount() build all the children (need that for render Appliance card),
+        // so it needs MemoryRouter's workaround for <Router /> element inside
+        const wrapper = mount(
+            <MemoryRouter keyLength={0}>
+                <AppliancePage />
+            </MemoryRouter>,
+            {attachTo: document.body}
+        );
+        let page = wrapper.find(AppliancePage);
+        const firstAppliance = page.find(Appliance).at(0);
+        const secondAppliance = page.find(Appliance).at(1);
+
+        const appliances = testData.storages.filter(appliance => {
+            return appliance.cluster === "false";
+        });
+
+        expect(page.state("selected_ids")).to.deep.equal([]);
+
+        firstAppliance.find("input").simulate("click");
+
+        expect(page.state("selected_ids")).to.deep.equal([appliances[0].id]);
+
+        secondAppliance.find("input").simulate("click");
+
+        expect(page.state("selected_ids")).to.deep.equal([appliances[0].id, appliances[1].id]);
+
+        firstAppliance.find("input").simulate("click");
+
+        expect(page.state("selected_ids")).to.deep.equal([appliances[1].id]);
+    });
+
+    it("Should update a list of selected clusters", () => {
+        // mount() build all the children (need that for render Appliance card),
+        // so it needs MemoryRouter's workaround for <Router /> element inside
+        const wrapper = mount(
+            <MemoryRouter keyLength={0}>
+                <AppliancePage />
+            </MemoryRouter>
+        );
+        wrapper.find("#pageButtonConfigured").simulate("click");
+        wrapper.update();
+
+        let page = wrapper.find(AppliancePage);
+        const firstCluster = page.find(Appliance).at(0);
+        const secondCluster = page.find(Appliance).at(1);
+
+        const clusters = testData.storages.filter(appliance => {
+            return appliance.cluster === "true";
```

Comment:
What's the reason `boolean` inside the `string`?

Reply:
Actually, it's a problem not only with a test, but also with the way how we store data. 
Will fix in next PR

Reply:
OK